README.md

```diff
@@ -2,7 +2,7 @@
 # My Portfolio
 
 > This is a Portfolio setup and mobile version  project by microverse to test the use of Flexbox for Positioning and also images and Background,
-![portfolio](./img/my-portfolio.jpg)
+![portfolio](./images/my-portfolio.jpg)
 
 
```

Comment:
- [ ] [Optional] Live link in your Readme is outdated and it doesn't reflect your new changes. You can update it. 😄